src/components/Dashboard/Tiles/Apps/AppTile.tsx

```diff
@@ -26,23 +26,34 @@ export const AppTile = ({ className, app }: AppTileProps) => {
   function Inner() {
     return (
       <>
-        <Text align="center" weight={500} size="md" className={classes.appName}>
-          {app.name}
-        </Text>
-        <Center style={{ height: '85%', flex: 1, width: '85%' }}>
+        <Stack
+          m={0}
+          p={0}
+          spacing="xs"
+          justify="space-around"
+          align="center"
+          style={{ height: '100%', width: '100%' }}
+        >
+          <Box hidden={false}>
```

Comment:
Here: Maybe add an option in the appearance settings to hide the app title if the user wants to